lib/reline/config.rb

```diff
@@ -29,18 +29,20 @@ class InvalidInputrc < RuntimeError
   attr_accessor :autocompletion
 
   def initialize
-    @additional_key_bindings = {} # from inputrc
-    @additional_key_bindings[:emacs] = {}
-    @additional_key_bindings[:vi_insert] = {}
-    @additional_key_bindings[:vi_command] = {}
-    @oneshot_key_bindings = {}
+    @additional_key_bindings = { # from inputrc
+      emacs: Reline::KeyActor::Base.new,
+      vi_insert: Reline::KeyActor::Base.new,
+      vi_command: Reline::KeyActor::Base.new
+    }
+    @oneshot_key_bindings = Reline::KeyActor::Base.new
     @editing_mode_label = :emacs
     @keymap_label = :emacs
     @keymap_prefix = []
-    @key_actors = {}
-    @key_actors[:emacs] = Reline::KeyActor::Emacs.new
-    @key_actors[:vi_insert] = Reline::KeyActor::ViInsert.new
-    @key_actors[:vi_command] = Reline::KeyActor::ViCommand.new
+    @key_actors = {
```

Comment:
Based on its usages, it feels like we can just call it `@key_bindings` or `@default_key_bindings` 🤔 
So we'll have (correct me if I'm wrong)
- `@key_bindings` (defined by Reline)
- `@additional_key_bindings` (defined by users)
- `oneshot_key_bindings` (added for dialogs?)

Reply:
Nice catch! I renamed it to `@default_key_bindings`.